app/controllers/api/api.js

```diff
@@ -18,4 +21,98 @@ function authenticatedUser(req, res, next) {
 
 router.get('/secret', authenticatedUser, function (req, res, next) {
   res.json({message: "secret"});
-});
\ No newline at end of file
+});
+
+// INDEX
+router.get('/api/helpz', function (req, res){
+  Service.find({}).sort({createdOn: -1}).exec(function (err, services) {
+    if (err) {
+      res.status(422).json({
+        success: false,
+        message: 'Could not find any Services b/c: ' + err
+      });
+    } else {
+      res.json(services);
+    }
+  });
+})
+
+// SHOW
+router.get('/api/helpz/:id', function (req, res){
+  Service.findById(req.params.id, function (err, service){
+    if (err) {
+      res.status(422).json({
```

Comment:
also, don’t need to use `status(422)` for everything, it means `unprocessable entity`, as in the json that comes in has problems, you also have 403 - forbidden, 404 - not found, etc